styles/google/catppuccin.user.css

```
@@ -0,0 +1,1093 @@
+        @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; }
+    }
+
+    #catppuccin(@lookup) {
+        @flamingo: @catppuccin[@@lookup][@flamingo];
+        @pink: @catppuccin[@@lookup][@pink];
+        @mauve: @catppuccin[@@lookup][@mauve];
+        @red: @catppuccin[@@lookup][@red];
+        @maroon: @catppuccin[@@lookup][@maroon];
+        @peach: @catppuccin[@@lookup][@peach];
+        @yellow: @catppuccin[@@lookup][@yellow];
+        @green: @catppuccin[@@lookup][@green];
+        @teal: @catppuccin[@@lookup][@teal];
+        @sky: @catppuccin[@@lookup][@sky];
+        @sapphire: @catppuccin[@@lookup][@sapphire];
+        @blue: @catppuccin[@@lookup][@blue];
+        @lavender: @catppuccin[@@lookup][@lavender];
+        @text: @catppuccin[@@lookup][@text];
+        @subtext1: @catppuccin[@@lookup][@subtext1];
+        @subtext0: @catppuccin[@@lookup][@subtext0];
+        @overlay2: @catppuccin[@@lookup][@overlay2];
+        @overlay1: @catppuccin[@@lookup][@overlay1];
+        @overlay0: @catppuccin[@@lookup][@overlay0];
+        @surface2: @catppuccin[@@lookup][@surface2];
+        @surface1: @catppuccin[@@lookup][@surface1];
+        @surface0: @catppuccin[@@lookup][@surface0];
+        @base: @catppuccin[@@lookup][@base];
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        
+        background: @base !important;
+        --uv-styles-color-surface: @crust;
+        --uv-styles-color-text-emphasis: @text;
+        --uv-styles-color-icon-on-secondary: @subtext0;
+            
+        .gb_Ka.gb_f {
+            background-color: @base !important;
+        }
+        .gb_Na.gb_f {
+            background-color: @base !important;
+        }
+        .uU7dJb {
+            border-bottom: 1px solid @surface0;
+            background-color: @mantle;
+            color: @subtext1;
+        }
+        .KxwPGc {
+            background-color: @mantle;
+        }
+        .lJ9FBc input[type="submit"] {
+            color: @text;
+            background-color: @surface0;
+            border: 1px solid @surface1;
+        }
+        .lJ9FBc input[type="submit"]:hover {
+            color: @text;
+            background-color: @surface0;
+            border: 1px solid @surface2;
+        }
```

```suggestion
        .lJ9FBc input[type="submit"] {
            color: @text;
            background-color: @surface0;
            border: 1px solid @surface1;
            &:hover {
                color: @text;
                background-color: @surface0;
                border-color: @surface2;
            }
        }
```
You can nest the two with the `&` selector. Note that I also changed the `border: ... @surface2;` to a `border-color: @surface2`.